Sources/PostgresNIO/New/Data/String+PSQLCodable.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 12.01.21.
+//
+
+import struct Foundation.UUID
```

Comment:
Ditch Xcode template headers.
```suggestion
import struct Foundation.UUID
```